pkg/utils/dbinstance/gsql.go

```diff
@@ -256,3 +289,16 @@ func determineGsqlPort(instance *sqladmin.DatabaseInstance) string {
 
 	return "-"
 }
+
+// disabledTokenSource is a mocked oauth token source for local testing.
+type disabledTokenSource struct{}
+
+// Token issues a mocked bearer token for local testing.
+func (ts *disabledTokenSource) Token() (*oauth2.Token, error) {
+	return &oauth2.Token{
+		AccessToken:  "let-me-in-pls",
```

Comment:
Given the mock isn't verifying the bearer token etc, might as well leave these as empty strings.